spec/support/statistics_helpers.rb

```diff
@@ -0,0 +1,245 @@
+module StatisticsHelpers
+
+  def gds_api_adapters_response_for_apply_uk_visa
+    {
+        'search_terms': gds_api_adapters_response_for_search_terms,
+        'searches': gds_api_adapters_response_for_searches,
+        'page_views': gds_api_adapters_response_for_page_views,
+        'problem_reports': gds_api_adapters_response_for_problem_reports
+    }
+  end
+
+  def gds_api_adapters_response_for_multipart_artefact
+    {
+        'search_terms': gds_api_adapters_response_for_search_terms,
+        'searches': gds_api_adapters_response_for_multipart_searches,
+        'page_views': gds_api_adapters_response_for_multipart_page_views,
+        'problem_reports': gds_api_adapters_response_for_multipart_problem_reports
+    }
+  end
+
+  def gds_api_adapters_response_with_no_performance_data
+    {}
+  end
+
+  def gds_api_adapters_response_for_searches
+    {
+        "data": [
+            {
+                "pagePath": "/apply-uk-visa",
+                "values": [
```

Comment:
Could we change all the fixtures so that some of them (if not all) have more than one entry in these `values` arrays to make sure that we are parsing them properly.  We have the `for_multipart` variants to make sure we're parsing multiple values in the `data` array properly.  I don't think we'd need separate fixtures for responses with one value vs multiple values though just changing the ones we already have.